server/src/main/resources/logback.xml

```diff
@@ -1,6 +1,8 @@
 <configuration>
+  <property name="WORKING_DIRECTORY" value="/work" />
+
   <appender name="FILE" class="ch.qos.logback.core.FileAppender">
-    <file>application.log</file>
+    <file>${WORKING_DIRECTORY}/welder.log</file>
```

Comment:
Think the `fileNamePattern` below needs updating too?

Reply:
thanks!..updated